md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ArchiveConsumpQueueAPI.java

```diff
@@ -321,6 +311,7 @@ RestWrapper insert(@ModelAttribute("acq")
             restWrapper = new RestWrapper(archiveConsumpQueue, RestWrapper.OK);
             LOGGER.info("Record with ID:" + archiveConsumpQueue.getQueueId() + " inserted into ArchiveConsumpQueue by User:" + principal.getName() + archiveConsumpQueue);
         } catch (Exception e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1166)